openupgradelib/openupgrade_merge_records.py

```diff
@@ -323,7 +323,7 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         if not field.store or field.compute or field.related:
             continue  # don't do anything on these cases
         op = field_spec.get(field.name, False)
-        l = all_records.mapped(field.name)
+        l = all_records.filtered('name').mapped(field.name)
```

Comment:
In any case, this should be `filtered(field.name)`, not `filtered('name')`

Reply:
thanks!
changed